package.json

```diff
@@ -11,7 +11,9 @@
   "engines": {
     "node": ">=0.4.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "extend": "^3.0.0"
```

Comment:
I'd prefer keeping this dependency free since we only need a very basic `extend()` implementation.